sdkconfig.defaults

```diff
@@ -96,6 +124,20 @@ CONFIG_SHA_BADGE_EINK_DEPG0290B1=y
 # CONFIG_SHA_BADGE_MPR121_DEBUG is not set
 CONFIG_SHA_BADGE_MPR121_HARDCODE_BASELINE=y
 # CONFIG_SHA_BADGE_EINK_DEBUG is not set
+# CONFIG_SHA_BADGE_EINK_LUT_DEBUG is not set
+# CONFIG_SHA_BADGE_POWER_DEBUG is not set
+
+#
+# SHA2017 Badge First Run
+#
+# CONFIG_DEBUG_ADD_DELAYS is not set
+# CONFIG_BASVS_BROKEN_BADGE is not set
+CONFIG_ALLOW_BATTERY_FIRST_BOOT=y
```

Comment:
Oh whoops!
How did this end up here?
I build a reference file from the components defaults..
Thanks for checking :)